### PR TITLE
Home page completed

### PR DESCRIPTION
**Pull request by IvanShterev**

### CODE REVIEW

File: ionic-angular-template/src/app/home/home.page.ts [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
+  private loggedUserSubscription: Subscription;
+  public loggedUser: any;
+  constructor(
+    private authService: AuthService,
+    private router: Router,
+    // private notificationService: NotificationService
+     ) {}
 
+  ngOnInit(): void {
+    this.loggedUserSubscription = this.authService.loggedUserData$.subscribe(async (data) => this.loggedUser = await data);
```
Comment (ochikov) [inline ionic-angular-template/src/app/home/home.page.ts]: I don't like this subscription. Consider refactor this code to something like:

```
this.authService.loggedUserData(async (userData) => {
do something with the data....
})
```

File: ionic-angular-template/src/services/auth.service.ts [hunk 2]
```diff
@@ -89,17 +87,17 @@ export class AuthService {
...
   }
 
-  async getUserDataIfAuthenticated() {
-    const loggedUser = JSON.parse(this.storageService.getItem('user'));
-    if (loggedUser) {
-      return (await firebase.firestore().collection('users').doc(`${loggedUser.uid}`).get()).data();
-    }
-    return null;
-  }
+  // async getUserDataIfAuthenticated() {
```
Comment (ochikov) [inline ionic-angular-template/src/services/auth.service.ts:90]: Why this fragment is commented.

Comment (IvanShterev) [reply ionic-angular-template/src/services/auth.service.ts:90]: I used it to initialize the subject but replaced it with "null".

Comment (ochikov) [reply ionic-angular-template/src/services/auth.service.ts:90]: Let's delete the commented code!

**Pull request merged**